### PR TITLE
Css bottom fixed

### PR DESCRIPTION
**Pull request by Aukse1976**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/position/terms/bottom/bottom.md [hunk 1]
```diff
@@ -42,6 +42,9 @@ Set the position of `.box` element `40px` from the bottom of the nearest relativ
   bottom: 40px;
 }
 ```
+![Shows use CSS bottom attribute with element position set to absolute](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-absolute-bottom.png)
```
Comment (dakshdeepHERE) [inline content/css/concepts/position/terms/bottom/bottom.md:45]:
```suggestion
![Shows the use of CSS bottom attribute with element position set to absolute](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-absolute-bottom.png)
```

File: content/css/concepts/position/terms/bottom/bottom.md [hunk 2]
```diff
@@ -56,3 +59,22 @@ Set the position of `.box` element `40px` from the elements bottom edge.
   bottom: 40px;
 }
 ```
+![Shows use CSS bottom attribute with element position set to relative](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-relative-bottom.png)
```
Comment (dakshdeepHERE) [inline content/css/concepts/position/terms/bottom/bottom.md:62]:
```suggestion
![Shows the use of CSS bottom attribute with element position set to relative](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-relative-bottom.png)
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Some changes and can you explain why you have added other two images if you are not using them in the entry?

File: content/css/concepts/position/terms/bottom/bottom.md (hunk 2 above)
Comment (Aukse1976) [reply content/css/concepts/position/terms/bottom/bottom.md]: Thanks :) Changes made

**Comment by dakshdeepHERE** [comment]
@Aukse1976 remove the other two identical images. Only keep the ones you are using the entry.

**Comment by Aukse1976** [comment]
> Some changes and can you explain why you have added other two images if you are not using them in the entry?

Two pairs of images are similar, but  file names are different.   I’ve only attached two with file names i thought are more 
suitable. And deleated those unused filed from 'media' folder.

**Comment by dakshdeepHERE** [comment]
I'm saying deleted these two files `media/css-bottom-position-relative.png` and `media/css-bottom-position-absolute.png`. As it is just redundant keep it there if they are not getting used anywhere.

**Comment by Aukse1976** [comment]
> I'm saying delete these two files `media/css-bottom-position-relative.png` and `media/css-bottom-position-absolute.png`. As it is just redundant keep it there if they are not getting used anywhere.

aukse@DESKTOP-AK0M9HI MINGW64 /c/projects/docs/media (css-bottom-fixed)
$ git commit -m "Deleated files: css-bottom-position-absolute.png and css-bottom-position-relative"
fatal: Unable to create 'C:/projects/docs/.git/index.lock': File exists.

Another git process seems to be running in this repository, e.g.
an editor opened by 'git commit'. Please make sure all processes
are terminated then try again. If it still fails, a git process
may have crashed in this repository earlier:
remove the file manually to continue.

**Comment by Aukse1976** [comment]
> I'm saying delete these two files `media/css-bottom-position-relative.png` and `media/css-bottom-position-absolute.png`. As it is just redundant keep it there if they are not getting used anywhere.

Please review  all changes are made in new branch "css-bottom" :) #4257 I hope it will help

**Pull request closed without merging**